recipe_scrapers/innit.py

```diff
@@ -0,0 +1,11 @@
+from ._abstract import AbstractScraper
+
+"""
+    Note that innit hosts recipes for several companies.  I found it while looking at centralmarket.com
+"""
+
+
+class Innit(AbstractScraper):
```

Comment:
Would you mind adding in the other accessor methods as per 80fd5e3bc20062ffbf06386feb42924f4e16eac0?

You can use #176 for reference!

Reply:
Added wrapper methods.  Will follow this pattern going forward.  Thanks for keeping the code stylish.  :)